ClassMoodApp/static/tsc/controllers/CreatePollingQuestionController.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path="../app.ts" />
+declare var angular: ng.IAngularStatic;
+module ClassMoodApp {
+    "use strict";
+    export class CreatePollingQuestionController {
+        static $inject = ["$scope", "$http", "$window", "$routeParams"];
+        public questions:Array<CreatePollingQuestionModel>;
+        public lecture_id: number;
+        public isStudent: boolean;
+        private userId: number;
+        private classId: number;
+        constructor(private $scope: ng.IScope,
+                    private $http: ng.IHttpService,
+                    private $window: ng.IWindowService,
+                    private $routeParams: ng.route.IRouteParamsService) {
+                        this.$http = $http;
+                        this.$scope = $scope;
+                        this.$window = $window;
+                        let curLink = $window.location.pathname.split("/");
+                        this.classId = parseInt(curLink[curLink.length-3]);
+                        this.lecture_id = parseInt(curLink[curLink.length-2]);
+                    }
+
+        public submitPollingQuestion(): void {
+		let text = this.$scope.question_text;
```

Comment:
Why is scope being used here? I'm using you are using ng-model and ng-controller for your html. If that's the case, there is no need to use scope. You can instead directly use the instance variables.